ksql-streams/pom.xml

```diff
@@ -53,6 +53,11 @@
             <artifactId>connect-runtime</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.confluent.support</groupId>
+            <artifactId>support-metrics-common</artifactId>
```

Comment:
I'm not sure it's worth the dependency just for a DI `Clock` type.  Maybe just use `Supplier<Long>` or add a `Clock` type to our common module...